solutions/secure/README.md

```diff
@@ -0,0 +1,11 @@
+# Cloud Object Storage for IBM Cloud - Financial Services Cloud solution
+
+This architecture creates an Cloud Object Storage(COS) and bucket on IBM Cloud® and supports provisioning the following resources:
+
+- A resource group, if one is not passed in.
+- A Cloud Object Storage(COS) for IBM Cloud, set up with
+    - KMS encryption to encrypt data
+    - Monitoring and Activity Tracking to capture information regarding the events in the COS.
```

Comment:
make sure its clear that the DA only configures bucket monitoring and AT for existing observability instances, that it doest create these instances